pom.xml

```diff
@@ -209,14 +213,6 @@
           <artifactId>maven-dependency-plugin</artifactId>
           <version>3.6.1</version>
         </plugin>
-        <plugin>
```

Comment:
Rely on fmt-maven-plugin define in java-shared-config: https://github.com/googleapis/java-shared-config/blob/cd05f0cad6fafc470d095c6ac428b87282c219fc/pom.xml#L221-L235